projects/packages/jetpack-mu-wpcom/src/features/calypso-locale-sync/sync-wp-admin-to-calypso.php

```diff
@@ -27,32 +30,33 @@ function update_calypso_locale( $locale ) {
 
 /**
  * Sync locale updated via /wp-admin/profile.php to Calypso.
- *
- * @param array   $meta   Meta values and keys for the user.
- * @param WP_User $user   User object.
- * @param boolean $update Whether the user is being updated rather than created.
  */
-function sync_wp_admin_locale_to_calypso( $meta, $user, $update ) {
-	$locale            = $meta['locale'];
-	$old_locale        = get_user_locale( $user );
-	$is_user_connected = ( new Connection_Manager( 'jetpack' ) )->is_user_connected();
+function sync_wp_admin_locale_on_profile_update() {
+	$user_id = get_current_user_id();
+	// phpcs:ignore WordPress.Security.NonceVerification.Missing
+	$locale = isset( $_POST['locale'] ) ? sanitize_text_field( wp_unslash( empty( $_POST['locale'] ) ? 'en' : $_POST['locale'] ) ) : null;
```

Comment:
Could this line be broken up a bit? I think it's fine to have a few if cases to make it more readable in the future.

Reply:
Sure 👍

Reply:
How about this?

```
$locale = null;

if ( isset( $_POST['locale'] ) ) {
    $locale = sanitize_text_field( wp_unslash( $_POST['locale'] ) );
}

if ( empty( $locale ) ) {
    $locale = 'en';
}
```

Reply:
Thanks!